services-js/commissions-app/src/pages/commissions/apply.tsx

```diff
@@ -11,52 +13,64 @@ export interface Props {
   commissions: Commission[];
 }
 
-export default class IndexPage extends React.Component<Props> {
+export interface State {
+  emailMatch: boolean;
+}
+
+export default class IndexPage extends React.Component<Props, State> {
   static async getInitialProps(): Promise<Props> {
     const commissions = await fetchCommissions();
     return { commissions };
   }
 
-  fullName: HTMLInputElement | null = null;
+  constructor(props) {
+    super(props);
+    this.state = {
+      emailMatch: true,
+    };
+    this.handleBlur = this.handleBlur.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
 
+  fullName: HTMLInputElement | null = null;
   middleName: HTMLInputElement | null = null;
-
   lastName: HTMLInputElement | null = null;
-
-  Address: HTMLInputElement | null = null;
-
+  address: HTMLInputElement | null = null;
   unit: HTMLInputElement | null = null;
-
   where: HTMLInputElement | null = null;
-
   city: HTMLInputElement | null = null;
-
   zip: HTMLInputElement | null = null;
-
   tel: HTMLInputElement | null = null;
-
   phone: HTMLInputElement | null = null;
-
   email: HTMLInputElement | null = null;
-
   confirmEmail: HTMLInputElement | null = null;
 
+  handleBlur() {
+    if (this.email === null || this.confirmEmail === null) {
+      return;
+    }
+
+    if (this.email.value != this.confirmEmail.value) {
```

Comment:
You can actually simplify all this.

Since you want `emailMatch` to be `true` when `this.email.value === this.confimEmail.value` you can replace this `if` statement with just:

```
this.setState({
  emailMatch: this.email.value === this.confimEmail.value,
});
```

LMK if that doesn’t make sense and we can work through it.